.github/workflows/main.yml

```diff
@@ -200,6 +200,20 @@ jobs:
       env:
         RUST_BACKTRACE: 1
 
+  rebuild_peephole_optimizers:
+    name: Rebuild Peephole Optimizers
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+      with:
+        submodules: true
+    - name: Build
+      run: |
+        cd cranelift && cargo build --features 'cranelift-codegen/rebuild-peephole-optimizers'
+    - name: Test
+      run: |
+        cd cranelift && cargo test --features 'cranelift-codegen/rebuild-peephole-optimizers'
```

Comment:
This doesn't check if the peephole optimizers were up to date. Only that they can be rebuild.